05-classification.Rmd

```diff
@@ -322,14 +332,14 @@ perim_concav + geom_point(aes(x = new_point[1], y = new_point[2]),
 So instead of just using the one nearest neighbour, we can consider several
 neighbouring points, say $K = 3$, that are closest to the new red observation
 to predict its diagnosis class. Among those 3 closest points, we use the
```

Comment:
```suggestion
To improve our prediction we can consider several neighbouring points, say $K = 3$, 
that are closest to the new red observation to predict its diagnosis class, 
instead of just using the one nearest neighbour. 
Among those 3 closest points, we use the
```

Reply:
edited